### ISSUE TITLE
 Expected CNameDeclaratorNode or CFuncDeclaratorNode, got Cython.Compiler.Nodes.CPtrDeclaratorNode 

### ISSUE THREAD

**Issue by ionelmc** [MarcoGorelli/cython-lint#8]
So I've added the pre-commit hook in https://github.com/ionelmc/python-hunter but I get this:

```
Traceback (most recent call last):
  File "/home/ionel/.cache/pre-commit/repoejaiyjhm/py_env-python3.10/bin/cython-lint", line 8, in <module>
    sys.exit(main())
  File "/home/ionel/.cache/pre-commit/repoejaiyjhm/py_env-python3.10/lib/python3.10/site-packages/cython_lint.py", line 355, in main
    ret |= _main(content, path)
  File "/home/ionel/.cache/pre-commit/repoejaiyjhm/py_env-python3.10/lib/python3.10/site-packages/cython_lint.py", line 270, in _main
    names, imported_names, ret = _traverse_file(
  File "/home/ionel/.cache/pre-commit/repoejaiyjhm/py_env-python3.10/lib/python3.10/site-packages/cython_lint.py", line 223, in _traverse_file
    ret |= visit_funcdef(node, filename, lines, violations=violations)
  File "/home/ionel/.cache/pre-commit/repoejaiyjhm/py_env-python3.10/lib/python3.10/site-packages/cython_lint.py", line 106, in visit_funcdef
    err_msg(
  File "/home/ionel/.cache/pre-commit/repoejaiyjhm/py_env-python3.10/lib/python3.10/site-packages/cython_lint.py", line 52, in err_msg
    raise CythonLintError(
cython_lint.CythonLintError: Unexpected error, please report bug. Expected CNameDeclaratorNode or CFuncDeclaratorNode, got <Cython.Compiler.Nodes.CPtrDeclaratorNode object at 0x7f1f8ba45090>
<Cython.Compiler.Nodes.CPtrDeclaratorNode object at 0x7f1f8ba45090>
```

**Comment by MarcoGorelli** [comment]
Nice, thanks a tonne for the report! I'll take a look this evening

**Comment by MarcoGorelli** [comment]
Should be fixed in version 0.1.8 - could you upgrade and try again please?

Thanks again for the report @ionelmc 🙌
